genai-perf/genai_perf/main.py

```diff
@@ -46,6 +50,31 @@ def run():
     else:  # profile
         args.func(args, extra_args)
 
+    if getattr(args, "enable_prometheus", False):
+        # Fix: This doesn't actually get logged.
```

Comment:
You can fix this by adding main to `logger.py`

Reply:
Ah, I didn't know that. Thanks, Brian! Done.